docs/src/content/docs/tr/guides/authoring-content.md

```diff
@@ -0,0 +1,209 @@
+description: Starlight'ın desteklediği Markdown sözdizimine genel bakış.
+---
+
+Starlight, `.md` uzantılı dosyalarda [Markdown](https://daringfireball.net/projects/markdown/) sözdizimini, [YAML](https://dev.to/paulasantamaria/introduction-to-yaml-125f) önbölümünde olduğu gibi başlık ve açıklama gibi metaverileri tanımlamak için destekler.
+
+Markdown desteği ve kullanımı farklılık gösterebileceğinden dolayı, bu dosya formatlarını kullanıyorsanız [MDX dokümantasyonu](https://mdxjs.com/docs/what-is-mdx/#markdown) ya da [Markdoc dokümantasyonu](https://markdoc.dev/docs/syntax)'nu incelediğinizden emin olun.
+
+## Satır İçi Stiller
+
+Metin **kalın**, _italik_ ya da ~~üstü çizili~~ olabilir.
+Text can be **bold**, _italic_, or ~~strikethrough~~.
+
+```md
+Metin **kalın**, _italik_ ya da ~~üstü çizili~~ olabilir.
+```
+
+Başka bir sayfaya [bağlantı ekleyebilirsiniz](/getting-started/).
+
+```md
+Başka bir sayfaya [bağlantı ekleyebilirsiniz](/getting-started/).
+```
+Kesme işaretleri ile `satır için kodu` vurgulayabilirsiniz.
+
+```md
+Kesme işaretleri ile `satır için kodu` vurgulayabilirsiniz.
+```
+
+## Görseller
+
+Starlight'ta görseller [Astro'nun kurulu optimize edilen dosya desteği](https://docs.astro.build/en/guides/assets/) ile kullanılır.
+
+Markdown ve MDX, ekran okuyucular ve yardımcı teknolojilere yönelik alternatif metin içeren görselleri göstermek için Markdown sözdizimini destekler.
+
+!["astro" metni içeren gezegen ve yıldızlar görseli](https://raw.githubusercontent.com/withastro/docs/main/public/default-og-image.png)
+
+```md
+!["astro" metni içeren gezegen ve yıldızlar görseli](https://raw.githubusercontent.com/withastro/docs/main/public/default-og-image.png)
+```
+
+Ayrıca, yerel olarak projenizde barındırılan görseller için ilişkili görsel dizin yolları desteklenir.
+
+```md
+// src/content/docs/page-1.md
+
+![Uzayda bir roket](../../assets/images/rocket.svg)
+```
+
+## Başlıklar
+
+Başlık kullanarak içerik yapınızı kurabilirsiniz. Markdown'daki başlıklar `#` sayısı ile satır başında oluşturulabilir.
+
+### Starlight'ta sayfa içeriği yapısı nasıl kurulur
+
+Starlight, sayfa başlığınızı en üst seviye başlık olarak kullanılacak şekilde yapılandırılmıştır ve içerik tablosunda "Genel Bakış" olarak yer alacaktır. Her sayfanın bir paragraf metniyle ve sayfa üstü başlığının `<h2>` ve alt seviyelerini kullanarak oluşturulmasını öneriyoruz:
+
+```md
+---
+title: Markdown Rehberi
+description: Starlight'ta Markdown nasıl kullanılır
+---
+
+Bu sayfa Starlight'ta Markdown'un nasıl kullanıldığını açıklar.
+
+## Satır İçi Stiller
+
+## Başlıklar
+```
+
+### Otomatik Başlık Bağlantıları
+
+Markdown'da başlık kullanmak, otomatik olarak başlıklar için bağlantı oluşturur. Böylece sayfanızın belli bölümlerini direkt olarak bağlantılandırabilirsiniz:
+
+```md
+---
+title: Sayfa İçeriğim
+description: Starlight'ın kurulu bağlantıları nasıl kullanılır
+---
+
+## Giriş
+
+[Görüşümü](#conclusion) aynı sayfanın aşağısına iliştirebilirim.
```

Comment:
Subtle, but the `#` link in this example is supposed to point to the heading below so should also be updated to match:

```suggestion
[Görüşümü](#görüş) aynı sayfanın aşağısına iliştirebilirim.
```

Reply:
Hi @delucis , Turkish alphabet's special characters like` ö,ı,ğ,ü,ş,ç` might break the functionality to scroll to related section. So I can fix it by typing `gorus`.

By the way, we use english names for html selectors like id and classes even if the html page is Turkish - this is commonly used approach in Turkiye. Should I change the link's name to Turkish or keep them as English?

Reply:
In Starlight, the IDs are generated automatically from the text, so in this case the heading, of `Görüş` automatically gets an ID of `görüş`, so I think what I suggested would be most correct given we’re writing about Starlight specifically.

You can see this in action in the already translated Turkish pages — for example, on the [Elle Kurulum](https://starlight.astro.build/tr/manual-setup) page, the “İçerik Ekleme” heading automatically gets the `id` of `#i̇çerik-ekleme`.

Reply:
Ah, I see it. Thanks @delucis , gonna fix it.